test/analysis/test_scorer.py

```diff
@@ -68,12 +68,19 @@ def test_abstain_labels(self) -> None:
         results_expected = dict(accuracy=0.6)
         self.assertEqual(results, results_expected)
 
-        # Test abstain=-1
+        # Test abstain=-1 for gold
         scorer = Scorer(metrics=["accuracy"], abstain_label=-1)
         results = scorer.score(golds, preds, probs)
         results_expected = dict(accuracy=0.75)
         self.assertEqual(results, results_expected)
 
+        # Test abstain=-1 for preds and gold
+        abstain_preds = np.array([-1, -1, 1, 1, 0])
+        abstain_probs = np.array([0.5, 0.5, 0.9, 0.7, 0.4])
+        results = scorer.score(golds, abstain_preds, abstain_probs)
```

Comment:
No need to pass in probs here. They're optional, and you're only calculating accuracy in this scorer, which just requires golds and preds.